pool/web/view/page/page_admin_invitations.ml

```diff
@@ -193,19 +199,24 @@ module Partials = struct
       |> to_row ~classnames:[ "font-bold" ]
     in
     div
-      [ h3
+      [ p
           [ txt
               Pool_common.(
-                Utils.text_to_string language I18n.InvitationsStatistics)
+                Utils.text_to_string language I18n.InvitationsStatisticsIntro)
           ]
       ; p
           [ txt
               Pool_common.(
-                Utils.text_to_string language I18n.InvitationsStatisticsIntro)
+                Format.asprintf
+                  "%s %i"
+                  (Utils.text_to_string language I18n.FilterNrOfContacts)
+                  total_match_filter)
           ]
       ; (sent_by_count
          |> CCList.map (fun (key, value) ->
-           (to_string key, to_string value) |> to_row)
+           ( to_string key
+           , Format.asprintf "%s / %i" (to_string value) total_match_filter )
+           |> to_row)
          |> fun rows ->
          rows @ [ total ] |> table ~thead ~a:[ a_class [ "table"; "simple" ] ])
```

Comment:
I'd replace the table with a sentence as long as no invitation was sent.
Without a sent invitation the table is hard to interpret.

Additionally, the columns in the statistics section aren't aligned. The "basic" and "Assignment" statistics have other column width then "Invitation"